uber/site_sections/preregistration.py

```diff
@@ -254,9 +244,6 @@ def prereg_payment(self, session, payment_id, stripeToken):
                 if attendee.amount_extra:
                     attendee.amount_paid = attendee.total_cost
             session.add(group)
```

Comment:
Hmm, maybe it was only committing so that the email could be rendered properly.  On second thought I think this can go away.  Obviously we'll want to test a few "make a group prereg" cases, but I think you're right and this can stay deleted.